osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerReadyButton.cs

```diff
@@ -83,6 +96,16 @@ private void updateButtonText()
                 else
                     countdownRemaining = countdown.TimeRemaining - timeElapsed;
 
+                if (countdownRemaining.Seconds <= 10 && (lastTickPlayed == null || lastTickPlayed != countdownRemaining.Seconds))
```

Comment:
This code should probably not be in a method named `updateButtonText()`. At the least I'd do a rename of the method here to something more generic, because I'm not super sure it's worth pulling this out of this method into another...

Reply:
Will restructure this on top of https://github.com/ppy/osu/pull/17643, which creates a place this code can be executed.